e2e-tests/create-fuels.test.ts

```diff
@@ -0,0 +1,27 @@
+import { test, expect } from '@playwright/test';
+
+test('counter contract - increment function call works properly', async ({ page }) => {
+  await page.goto('http://127.0.0.1:3000/', { waitUntil: 'networkidle' });
+
+  await page.waitForTimeout(2000);
+
+  const topUpWalletButton = page.getByText('Top-up Wallet');
+  await topUpWalletButton.click();
+
+  const welcomeToFuelText = page.getByText('Welcome to Fuel');
+  await expect(welcomeToFuelText).toBeVisible();
+
+  await page.waitForTimeout(2000);
+
+  await page.reload();
```

Comment:
Unfortunately we need to reload the page after topping up the wallet because for some reason the balance isn't being updated properly as mentioned in #2048